src/compliance/compliance-requirements.controller.ts

```diff
@@ -266,4 +266,29 @@ export class ComplianceRequirementsController {
     const result = await this.complianceRequirementsService.modify(ticker, id, params);
     return handleServiceResult(result);
   }
+
+  @ApiOperation({
+    summary: 'Check if the requirements are paused',
+    description:
+      'This endpoint checks if the compliance requirements are paused for a given ticker',
+  })
+  @ApiParam({
+    name: 'ticker',
+    description: 'The ticker of the Asset whose compliance requirements status are to be fetched',
+    type: 'string',
+    example: 'TICKER',
+  })
+  @ApiOkResponse({
+    description: 'Compliance Requirement status',
+    schema: {
+      type: 'boolean',
+    },
+  })
+  @ApiNotFoundResponse({
+    description: 'The Asset does not exist',
+  })
+  @Get('are-paused')
+  public async areRequirementsPaused(@Param() { ticker }: TickerParamsDto): Promise<boolean> {
```

Comment:
```suggestion
  public async areRequirementsPaused(@Param() { ticker }: TickerParamsDto): Promise<ComplianceStatusModel> {
```
and declare it as a model.

Keep the scope to only being pause or not for now, but allow it to be extended and a bit more generic.